concourse/pipelines/monitoring.yml

```diff
@@ -5,7 +5,7 @@ resources:
   - icon: github
     name: govuk-infrastructure
     source:
-      branch: fix_grafana
+      branch: main
```

Comment:
Oops ;) Thanks for fixing. (I could tell you an [amusing story](https://twitter.com/0xakashb/status/485207945626787840?lang=en) on the theme of accidentally leaving temporary/test changes in at review time and them going in unnoticed 😀)